src/assets/icons/close-icon.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file Close Icon
+ * @requires lit
+ * @exports TemplateResult
+ * Path: src/assets/icons/close-icon.ts
+ */
+import { svg } from 'lit';
+
+export default svg`
+<svg
```

Comment:
Note that templates for full SVG images should still use an `html` template tag. The `svg` tag is meant for templating internal _children_ to be placed _inside_ of an SVG.

https://lit.dev/docs/api/templates/#svg

Reply:
Thanks @latonv!